iothub/device/src/Exceptions/ExceptionHandlingHelper.cs

```diff
@@ -11,7 +11,21 @@ namespace Microsoft.Azure.Devices.Client
 {
     internal class ExceptionHandlingHelper
     {
-        public static IDictionary<HttpStatusCode, Func<HttpResponseMessage, Task<Exception>>> GetDefaultErrorMapping()
+        /// <summary>
+        /// Mapper to map IoT hub service returned 6 digit error codes to <see cref="IotHubClientException.ErrorCode"/>.
+        /// </summary>
+        /// <param name="errorCode">The 6 digit error code returned by IoT hub service.</param>
```

Comment:
I am not sure if this field would always be 6 digit numeric codes. We do have cases of non-numeric codes returned by the hub on the service client side. Error message is like [this](https://github.com/Azure/azure-iot-sdk-csharp/blob/previews/v2/iothub/service/tests/ExceptionHandlingHelperTests.cs#L107).

Reply:
That is fair. However, since we don't know of any such case on the client side yet, we'll have to wait until we encounter this scenario and then address it.